src/libYARP_sig/include/yarp/sig/PointCloudNetworkHeader.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2006-2018 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * BSD-3-Clause license. See the accompanying LICENSE file for details.
+ */
+
+#ifndef YARP_SIG_POINTCLOUDNETWORKHEADER_H
+#define YARP_SIG_POINTCLOUDNETWORKHEADER_H
+
+#include <yarp/conf/system.h>
+
+#include <yarp/os/NetInt32.h>
+
+#include <yarp/sig/PointCloudTypes.h>
+
+namespace yarp {
+    namespace sig {
+        class PointCloudNetworkHeader;
+    }
+}
+
+
+// YARP_BEGIN_PACK
+/**
+ * @brief The yarp::sig::PointCloudNetworkHeader class
+ */
+class yarp::sig::PointCloudNetworkHeader
+{
+public:
+    PointCloudNetworkHeader() :    width(10),
```

Comment:
Here and the following lines the indentation seems somwhat wrong.

Reply:
It seems ok :thinking:

Reply:
This is how I see it.
<img width="418" alt="schermata 2018-03-15 alle 19 40 08" src="https://user-images.githubusercontent.com/14168673/37484432-beb210b6-2888-11e8-958e-27aacf8b597d.png">
There are extra spaces here and there after the default constructor semicolon 😄

Reply:
Ah ok I see, I thought was referred to the identation of public 😅